src/board/devices/input/rotarySwitch.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Represents the rotary switch in memory.
+ *
+ * @author Philippe Schneider
+ */
+
+import { Device } from 'board/devices/device'
+import { Byte, Word } from 'types/binary'
+
+export class RotarySwitch extends Device {
+  private RotarySwitch_ADDRESS = Word.fromUnsignedInteger(0x60000211)
```

Comment:
This could be `static readonly` and all in capital letters.

Reply:
Thank you, changed it.